chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -821,7 +821,7 @@ public synchronized void loadChunks(TaskTracker taskTracker, World world, Collec
       }
 
       for (RegionPosition region : regions) {
-        dimension.getRegion(region).parse(yMin, yMax);
+        ((JavaDimension) dimension).getRegion(region).parse(yMin, yMax);
```

Comment:
Since regions are handy for selecting many chunks, could we add virtual regions (= squares of many chunks) to Bedrock? As a UI feature, not as in actually loading regions, if the world format doesn't actually have regions.

Reply:
The idea being abstract out a "selection region" that also happens to be the same size as a java region?